arb_os/output.mini

```diff
@@ -217,7 +207,9 @@ public impure func emitLog(
 
     asm((
         0,
-        l1message,
+        l1message with {
+            msgData: unsafecast<ByteArray>(bytearray_marshalFull(l1message.msgData))  // because caller expects marshalled bytes
```

Comment:
Is there a good reason why we're not just using a ByteArray here when outputing a log?